config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  resources :games
   root 'game#index'
+
+  post 'new_game', to: 'game#new_game'
```

Comment:
`resources :games` will generate a route for you that corresponds to `games#create`, which is what you can use for this action instead of creating your custom route. See references here:

https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions